Mathlib/Data/List/Defs.lean

```diff
@@ -410,6 +410,12 @@ def replaceIf : List α → List Bool → List α → List α
   | l, [], _ => l
   | n :: ns, tf :: bs, e@(c :: cs) => if tf then c :: ns.replaceIf bs cs else n :: ns.replaceIf bs e
 
+/-- `repeatSelf n l` is one list conisting of l repeated n times. -/
```

Comment:
There was a [recent zulip discussion](https://leanprover.zulipchat.com/#narrow/channel/217875-Is-there-code-for-X.3F/topic/repeat.20list/with/514068338) about this particular definition: did you see this already? That thread suggests a slightly different name, and a different definition (`(List.replicate n l).flatten`) instead.